tuneme/migrations/0004_delete_polls_index_page.py

```diff
@@ -38,12 +38,10 @@ def delete_polls_index_page(apps, schema_editor):
         ct_pollsindexpage.delete()
 
 
-class Migration(migrations.Migration):
-    dependencies = [
-        ('tuneme', '0003_move_pages_to_index_pages'),
-    ]
+def remove_tables_from_postgres(apps, schema_editor):
+    db_type = schema_editor.connection.vendor
 
-    operations = [
+    if ((db_type == 'postgres') or (db_type == 'postgresql')):
```

Comment:
why wrap this in brackets? `if db_type == 'postgres' or db_type == 'postgresql'`

Reply:
Sorry, I was working in Javascript over the weekend . . .